lstchain/tools/lstchain_create_irf_files.py

```diff
@@ -28,6 +28,9 @@
 If you want to generate source-dependent IRFs, source-dep flag should be activated.
 The global alpha cut used to generate IRFs is stored as AL_CUT in the HDU header.
 
+Modified IRFs can be created in case of considering systematics in the
+light efficiency. This can be done by setting a value different to one for the
+"scale_true_energy" argument present in the DataBinning Component of the configuration file. 
```

Comment:
I suggest something like:
```
Modified IRFs with true energy scaled by a given factor can be created to evaluate the systematic uncertainty in the light collection efficiency. This can be done by setting a value different from one for the "scale_true_energy" argument present in the DataBinning Component of the configuration file of the IRF creation Tool.
```